go.mod

```diff
@@ -45,4 +45,6 @@ require (
 	k8s.io/api v0.20.4
 	k8s.io/apimachinery v0.20.4
 	k8s.io/client-go v0.20.4
+	k8s.io/klog v1.0.0
```

Comment:
i think we only want v2 here right?

Reply:
yeah, should work. v1 is needed only if one of the projects is still using v1, but I think it works without it as well.

Reply:
hmmm now I'm checking and it doesn't work LOL.